spec/requests/alerts_spec.rb

```diff
@@ -80,7 +80,7 @@
         "subcount"  => 1,
         "resources" => [
           {
-            "href" => api_alert_alert_action_url(nil, alert.compressed_id, alert_action.compressed_id)
+            "href" => api_alert_alert_action_url(nil, alert.id.to_s, alert_action)
```

Comment:
same question as above for all collection/subcollection _url calls.

Reply:
Yeah, these should be. Will update!